js/store.js

```diff
@@ -11,6 +10,91 @@ const store = {
       }
     }
     browser.runtime.onMessage.addListener((m) => store.messageHandler(m));
+
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList;
```

Comment:
Okay great suggestion thank you. Is there a way to consolidate the code below into a single line?

```javascript
    const { firstPartyWhiteList, thirdPartyWhiteList }
      = await this.getWhiteList();
    this.firstPartyWhiteList = firstPartyWhiteList;
    this.thirdPartyWhiteList = thirdPartyWhiteList;
```
I tried this,

```javascript
{ this.firstPartyWhiteList this.thirdPartyWhiteList } = await this.getWhiteList();
```
But I get a `SyntaxError: expected expression, got '='` message in the console...